src/Service/SettingsManager.php

```diff
@@ -104,6 +104,10 @@ public function get($name)
             case 'mercadopago_access_token':
                 $name = $this->isMercadopagoLivemode() ? 'mercadopago_live_access_token' : 'mercadopago_test_access_token';
                 break;
+            case 'mercadopago_access_token_for_test':
+                // to be used only for test
+                $name = !$this->isMercadopagoLivemode() ? 'mercadopago_live_access_token' : '';
```

Comment:
Does this work? I mean trying to retrieve a setting with an empty (`''`) name will return `null`, right?

Reply:
@alexsegura  It should never retrieve an empty string because the setting `mercadopago_access_token_for_test` should only be requested when Mercadopago isn't in live mode. I just added this conditional to be clear about to use this key only for test purpose. Let me know if it does not make sense for you.

Reply:
It's fine.